indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -118,7 +128,9 @@
                                 </td>
                                 {% for item in static_columns %}
                                     {% if item.id == 'state' %}
-                                        <td class="i-table" data-searchable="{{ abstract.public_state.title | lower }}">
+                                        {% set acceptances = abstract.reviews|selectattr('proposed_action.name', 'equalto', 'accept')|list|length %}
+                                        {% set sort_text = '{}-{}-{}'.format('1' if abstract.is_in_final_state else '0', abstract.state.name, acceptances) %}
```

Comment:
Does `abstract.is_in_final_state` here actually affect the sorting at all? You already sort by the state afterwards, and `is_in_final_state` only checks for `state != submitted`

Reply:
It can be useful to group abstracts in a final state together.

Reply:
Ah, what I originally meant when I suggested grouping them together was to not have "under review" mixed into the list of abstracts in some other state. Anyway, nothing wrong with explicitly having the under-review ones separated from the rest